lib/user.rb

```diff
@@ -0,0 +1,20 @@
+require 'pry'
+class User
+
+  attr_reader :name, :jokes
+
+  def initialize(name)
+    @name = name
+    @jokes = []
+  end
+
+  def learn_joke(joke)
```

Comment:
Keep a close eye on the interaction pattern - we wanted the following methods to be `learn` and `tell` - when given specific methods to create, create them exactly as asked!